File: camera_system/camera_system_stub/camera_system_facade/classes/PerSecondStats.py
class PerSecondStats:
    def __init__(self, camera_id, stat_time, num_tracked_people, has_saved_image):
        self.camera_id = camera_id
        self.stat_time = stat_time
        self.num_tracked_people = num_tracked_people
        self.has_saved_image = has_saved_image

    def to_json(self):
        return {
            "CameraId": self.camera_id,
            "DateTime": self.stat_time.strftime("%Y-%m-%d %H:%M:%S"),
            "NumTrackedPeople": self.num_tracked_people,
            "HasSavedImage": self.has_saved_image
        }

File: Back-End/WebAPI/JsonGenerator/Classes/PerSecondStats.cs
using System;

namespace JsonGenerator.Classes
{
    /// <summary>
    /// An object which represents information extracted for each second of real-time monitoring.
    /// </summary>
    public class PerSecondStats
    {
        #region Attributes 

        // Date and Time Information, representing the exact second represented by the statistics in this object.
        public long UnixTime { get; set; }

        // The ID of the camera which produced these statistics for this exact second.
        public int CameraId { get; set; } 

        // Statistic #1: Stores the number of people identified within the second.
        public int NumTrackedPeople { get; set; }

        // Statistic #2: Indicates if the embedded system has stored an image locally of this second. This happens when a key statistic has occured. 
        public bool HasSavedImage { get; set; }

        #endregion

        // Constructor with a flag of false by default for HasSavedImage.
        public PerSecondStats (int CameraId, long UnixTime, int NumTrackedPeople, bool HasSavedImage = false)
        {
            this.CameraId = CameraId;
            this.UnixTime = UnixTime;
            this.NumTrackedPeople = NumTrackedPeople;
            this.HasSavedImage = HasSavedImage;
        }
    }
}
